core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/BoxDerivedInfo.java

```diff
@@ -26,11 +31,28 @@
   @Temporal(TemporalType.TIMESTAMP)
   private Date lastModified;
 
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date created;
+
+  @OneToOne(targetEntity = UserImpl.class)
+  @JoinColumn(name = "creator", nullable = false)
+  @JsonBackReference
+  private User creator;
```

Comment:
A bunch of other user fields are annotated as such and I don't have a good testing strategy to figure out what's serialising these things. In other news, I'm now a fan of magical thinking.